### PR TITLE
Enhancement: Convert `main.py` to use `customtkinter`

### PR DESCRIPTION
**Pull request by dishb**
The first step in completing #29.

I have **not** tested these changes. Please test them yourself.

This PR includes changes that...
- convert `main.py` to use `customtkinter`
- some widgets like `ttk.Treeview()` were not converted due to lack of `customtkinter` counterparts.

### CODE REVIEW

**Comment by jacobranz** [comment]
Looking at this now

**Review by dishb — COMMENTED** [review]
@jacobranz - Not sure this exists in actual CustomTkinter... Maybe check?

File: main.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def show_frame(self, controller):
             self.frame[controller] = frame = controller(self.container, self)
             frame.grid(row=0, column=0, sticky="nsew")
         frame = self.frame[controller]
-        frame.tkraise()
+        frame.CTkraise()
```
Comment (dishb) [inline main.py:45]: Please check that `customtkinter.CTkFrame` has this function.

Comment (jacobranz) [reply main.py:45]: It does not unfortunately, I was getting errors and hoping that would fix it. But no luck.

Comment (dishb) [reply main.py:45]: 👍 Please revert that commit.

**Comment by jacobranz** [comment]
Current tests failing, will take a further look soon. Currently, working off of my password manager repo as that is my priority.

**Comment by dishb** [comment]
> Current tests failing, will take a further look soon. Currently, working off of my password manager repo as that is my priority.

Gotcha 👍

**Comment by jacobranz** [comment]
@dishb, would you like to pick this up again? Finally getting back to looking at this repo full time to make massive improvements.

**Comment by dishb** [comment]
Yeah, I'll create a new PR this weekend.

**Comment by jacobranz** [comment]
Perfect, getting ready to restructure the project out of just one file as well. Cheers!

**Pull request closed without merging**